api/src/main/java/org/pmiops/workbench/db/dao/projection/PrjUser.java

```diff
@@ -0,0 +1,98 @@
+package org.pmiops.workbench.db.dao.projection;
```

Comment:
If this code is generate, please add a generated comment with provenance

Reply:
Good idea.

With respect to the name `PrjUser` above, I was trying to draw a parallel with `DbUser`, but I like `ProjectedUser` better than `UserProjection`.